Roopan-Microsoft/psl-cwyd-main
# fix: CWYD | SFI Issue fix for Storage and Cosmos | rbac and keys deployment changes

## Purpose

* Updated code will fix the role-based authentication using managed identity. Now we can use storage and cosmos based on Auth type i.e. key or rabac.

## Does this introduce a breaking change?





## How to Test
*  Just deploy by using one click deploy button and select auth type as keys or rbac. The application should work for both key based and role-based authentication.

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

## Other Information
